lisa/tools/lscpu.py

```diff
@@ -97,7 +97,10 @@ def get_architecture(self, force_run: bool = False) -> str:
         return architecture
 
     def get_core_count(self, force_run: bool = False) -> int:
-        result = self.run(force_run=force_run)
+        if not self._core_count:
```

Comment:
Do you know the root cause? Can you share me the log?

Reply:
Is Lscpu tool guaranteed to run once before get_core_count is called? I had found in testing that it could return 0 if force_run wasn't passed on the first call.

Reply:
If so, it should raise an exception, if it doesn't pass on first time. So, we can see what's the error, then mitigate or retry.